bot/botUtils.js

```diff
@@ -33,10 +34,47 @@ exports.isAdmin = (member) => {
   return false;
 };
 
+// Checks if the member has been verified
+exports.isVerified = async (userId) => {
+  const memberData = await Member.findOne({ attributes: ['verified'], where: { discordUser: userId } });
+  if (memberData) {
+    return !!memberData.dataValues.verified;
+  }
+  return false;
+};
+
 // Update the user points in the database
 exports.getUserPointsandUpdate = async (userId, pointsToAdd) => {
-  const memberData = await Member.findAll({ attributes: ['points'], where: { discordUser: userId } });
-  await Member.update(
-    { points: memberData[0].dataValues.points + pointsToAdd },
-    { where: { discordUser: userId } });
+  const memberData = await Member.findAll({ attributes: ['points', 'verified'], where: { discordUser: userId } });
+  const { points, verified } = memberData[0].dataValues;
+  const pointsBeingAdded = parseFloat((points + pointsToAdd).toFixed(2));
+  verified ? await Member.update(
+    { points: pointsBeingAdded },
+    { where: { discordUser: userId } }) : util.log('User is not verified', userId, 4);
+};
+
+// Using async and await is the only way I could get this to work
+// It needed to be forced to wait and go line by line to get the proper data
+exports.welcomeCommand = async (member) => {
+  const { user } = member;
+  util.log('User id', user.id, 4);
+  let welcomeString = null;
+  const pointsMethods = `
+    **How to Earn Points:**
+    \n\`Chatting with friends\`
+    Chat with others and earn points! Get rewarded for talking to others!
+    \n\`Gaming, Streaming, and More!\`
+    Be rewarded for engagement through streaming, playing games with others, listening to Spotify, and more!
+    \n\`Invite New Members\`
+    Invite new members to the group and be rewarded with points!
```

Comment:
should there be anything here about only full sail students?

Reply:
Youre right there should be. I will change it to include something about Full Sail Students.

Reply:
other than that it looks good to me